controllers/secretagentconfiguration_controller.go

```diff
@@ -84,14 +85,21 @@ func (reconciler *SecretAgentConfigurationReconciler) Reconcile(req ctrl.Request
 
 	if instance.Spec.AppConfig.SecretsManager != v1alpha1.SecretsManagerNone {
 		var dir string
+		var cloudCredNs string
```

Comment:
kind of a nitpick but I think this should be `cloudCredNS` surprised go fmt/lint didn't catch that.